webapp-django/crashstats/crashstats/static/crashstats/css/base.less

```diff
@@ -1,5 +1,254 @@
-@import "reset.less";
-@import "variables.less";
-@import "mixins.less";
-@import "forms.less";
-@import "tables.less";
+/*
+Copyright (c) 2009, Yahoo! Inc. All rights reserved.
+Code licensed under the BSD License:
+http://developer.yahoo.net/yui/license.txt
+version: 3.0.0
+build: 1549
+*/
+
+@import "base/reset.less";
+@import "base/variables.less";
+@import "base/mixins.less";
+@import "base/forms.less";
+@import "base/tables.less";
+@import "base/typography.less";
+@import "base/layout.less";
+
+/* Layout elements */
+@import "components/nav.less";
+@import "components/footer.less";
+
+/* Common components */
+@import "components/accordion.less";
+@import "components/admin.less";
+@import "components/code.less";
+@import "components/message.less";
+@import "components/pagination.less";
+@import "components/panel.less";
+@import "components/simplebox.less";
+@import "components/sumo_link.less";
+@import "components/table_sorter.less";
+@import "components/tree.less";
+
+/* Other shared styles */
+a {
+    text-decoration: none;
+}
+a, a:link, a:hover, a:active {
+    color: #0033FF;
+    text-decoration: underline;
+}
+.ui-widget-content a:visited, /* Override jQuery UI's behavior */
+a:visited {
+    color: #800080;
+}
+
+body > h1 {
+    font-size: 200%;
+}
+
+.clear {
+    clear: both;
+}
+.clear_left {
+    clear: left;
+}
+.clear_right {
+    clear: right;
+}
+.hidden {
+    display: none;
+}
+.visually-hidden {
+    .accessibly-hidden()
+}
+
+
+#mainbody {
+    position: relative;
+    margin: 1em 2em;
+    text-align: left;
+    flex: 1;
+}
+.loading {
+    position: absolute;
+    top: 45%;
+    left: 45%;
+}
+.inline-loader {
+    width: 18px;
+    height: 18px;
+    vertical-align: middle;
+}
+
+
+
+/* --- */
```

Comment:
Below this line are the elements I could not verify exist on the site, and will eventually be removed in a later PR after some thorough site crawling/testing to ensure they really don't exist anywhere.